bench/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "sanctuary-type-classes-benchmarks",
+  "version": "0.0.0",
+  "private": true,
+  "license": "WTFPL",
```

Comment:
When I tried that the first time, npm told me: "Cannot install sanctuary-type-classes as a dependency of itself"